### PR TITLE
Dev hackathon 2018 nov

### PR DESCRIPTION
**Pull request by CITguy**
DO NOT MERGE

Using PR to compare differences with `master` branch. As updates are brought into `master`, this PR should show fewer and fewer files changed.

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Building with commit db3d1f63eabaf287c06eec688dd782f4f1fd3eee

https://app.netlify.com/sites/helix-ui/deploys/5c8c216b897ef70009e9f3ad

File: docs/_data/icons.json5 [hunk 1]
```diff
@@ -4,36 +4,36 @@
         minver: '0.1.10',
     },
     {
-        name: 'angle-bottom',
-        minver: '0.2.1',
+        name: 'angleBottom',
+        minver: '0.N.0',
```
Comment (CITguy) [inline docs/_data/icons.json5]: TODO: make sure to update the `minver` props when merging into `master`

File: docs/_data/icons.json5 [hunk 2]
```diff
@@ -575,4 +541,300 @@
             'person',
         ],
     },
+
+    /* DEPRECATED */
+    /* TODO: migrate to separate file */
```
Comment (CITguy) [inline docs/_data/icons.json5]: TODO: update icons vue demo to grab an additional file (if deprecated icons should be shown)

**Comment by CITguy** [comment]
Closing. Will keep branch around for reference when making pipeline updates.

**Pull request closed without merging**